internal/report/report.go

```diff
@@ -111,11 +111,10 @@ func Generate(w io.Writer, rpt *Report, obj plugin.ObjTool) error {
 		return printAssembly(w, rpt, obj)
 	case List:
 		return printSource(w, rpt)
-	case WebList:
-		return printWebSource(w, rpt, obj)
 	case Callgrind:
 		return printCallgrind(w, rpt)
 	}
+	// Note: WebList handling is in driver package.
```

Comment:
If WebList is the only "enum" that is defined but not handled, maybe we should print out this enum value in the following "expected output format"? To make it easier debug if this method ss accidentally called for WebList.